README.md

```diff
@@ -6,51 +6,43 @@ We run an [internal server](./internal-server.md) so private datasets can be kep
 
 We have have general experience [tracking neuroimaging data with git-annex](./git-annex.md).
 
-The data curation scripts should be placed inside the BIDS datasets, under the `code/` folder. See [BIDS specification](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#code). For more convenience, you can create a PR with a curation script that you are working on, so that others can give feedback; once the script is validated, you can simply close the PR and delete the branch without merging.
+# ⚠️ Every dataset should have the following files 
 
-Every dataset should have a README inside it. Example:
-~~~
-# <NAME OF DATASET>
-
-<PLEASE MODIFY>This is an MRI dataset acquired in the context of the XX project. It also contains manual segmentation of MS lesions from expert raters.
-
-Dataset shared by: <NAME AND EMAIL>
-<IF THERE WAS EMAIL COMM>Email communication: <DATE OF EMAIL AND SUBJECT>
-<REPOSITORY OF PROJECT/MODEL, etc>Repository: https://github.com/ivadomed/model_seg_ms_mp2rage
+```
+├── README.md
+├── participants.json
+├── participants.tsv
+├── dataset_description.json
+└── code
+    └── <script_you_used_for_dataset_curation>.py
+```
 
-<IF THERE ARE UPDATES IN THE DATASET:>2022-10-30: Added new data from INsIDER_SCT_Segmentations_COR
+For details, see [BIDS specification](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#code).
 
-## dataset structure
+## `README.md`
 
-participants.tsv: Include the following columns:
-- 'pathology': can take values listed in https://docs.google.com/spreadsheets/d/1yjcA8Z0COn4OZxusIDHjStH2DpeXvscsj-aWE2X-_sg/edit#gid=0
-- 'data_id' is the name center_study_subject used to locate the unprocessed data as under duke/mri/.
-- 'institution_id' is used to identify CenterStudy.
-- others? XXX (Jan pls continue)
+The `README.md` is a [markdown](https://markdown-guide.readthedocs.io/en/latest/index.html) file describing the dataset in more detail.
 
+<details><summary>README.md template:</summary>
 
-XXX Jan contnue -- asju julien to check afterward
+```
+# <NAME OF DATASET>
 
+This is an MRI dataset acquired in the context of the <XYZ> project. 
+It also contains manual segmentation of <MS lesions/tumors/etc> from <one/two/or more> expert raters. 
+Segmentation are located under the derivatives folder.
```

Comment:
For the record, today we had a good long discussion where we contrasted BIDS's `derivatives/` idea against Datalad's [YODA](https://handbook.datalad.org/en/latest/basics/101-127-yoda.html) idea. They are sort of opposite: Datalad suggests derived datasets should be in a separate repo _with a pointer_ back to their sources. BIDS says all derived data should live in.

@mguaypaq and I both lean towards the YODA model.

But I guess segmentations are a kind of data that's very specific and closely related to the source data. It would be awkward to keep them in a fully separate repo. So maybe segmentations get a pass. But in general I would discourage the use of the derivatives/ folder entirely.

Reply:
> But I guess segmentations are a kind of data that's very specific and closely related to the source data.

yes, precisely. In fact, in most cases, these are generated manually by radiologists, and as you rightly pointed out, are going in pair with the source data. I do agree though, that for _derived_ (ie: product of source data going through some sort of analysis pipeline), the YODA model is more attractive.